misk/src/test/kotlin/misk/client/MockWebServerService.kt

```diff
@@ -0,0 +1,27 @@
+package misk.client
+
+import com.google.common.util.concurrent.AbstractIdleService
+import okhttp3.mockwebserver.MockWebServer
+import java.io.File
+import javax.inject.Singleton
+
+@Singleton
+class MockWebServerService(val socketFile: String?) : AbstractIdleService() {
```

Comment:
should this be named to make it clear it uses a unix file socket? that would not be obvious to me